neos/Neos.NeosIo
# TASK: Improve button and image teaser styling

This change replaces the svg based cut corners of images and buttons
with css features.

The fallback for older browsers is just to have square images/buttons.

- [x] Tested in IE 11

How to test: 
* Check all the buttons on the homepage
* Check the show case items
